apps/voting/test/voting.js

```diff
@@ -238,6 +238,36 @@ contract('Voting App', accounts => {
         })
     })
 
+    context('wrong initializations', () => {
+        beforeEach(async() => {
+            const n = '0x00'
+            token = await MiniMeToken.new(n, n, 0, 'n', 0, 'n', true) // empty parameters minime
+        })
+
+        it('fails if min acceptance quorum is 0', () => {
+            const neededSupport = pct16(20)
+            const minimumAcceptanceQuorum = pct16(0)
+            return assertRevert(async() => {
+                await app.initialize(token.address, neededSupport, minimumAcceptanceQuorum, votingTime)
+            })
+        })
+
+        it('fails if min support is greater than 100', () => {
+            const neededSupport = pct16(20)
+            const minimumAcceptanceQuorum = pct16(50)
+            return assertRevert(async() => {
+                await app.initialize(token.address, neededSupport, minimumAcceptanceQuorum, votingTime)
+            })
+        })
+
+        it('fails if min acceptance quorum is greater than min support', () => {
```

Comment:
I think l255 and this line's description should be flipped?

Reply:
Ha, good catch, thanks!